compiler/rustc_mir/src/transform/check_consts/validation.rs

```diff
@@ -276,7 +277,7 @@ impl Validator<'mir, 'tcx> {
     }
 
     pub fn qualifs_in_return_place(&mut self) -> ConstQualifs {
-        self.qualifs.in_return_place(self.ccx)
+        self.qualifs.in_return_place(self.ccx, self.error_emitted)
```

Comment:
ah and then we have to change the `error_emitted` field to contain an `Option<ErrorReported>`, too